fastai/vision/image.py

```diff
@@ -432,7 +432,8 @@ def show_image(img:Image, ax:plt.Axes=None, figsize:tuple=(3,3), hide_axis:bool=
                 alpha:float=None, **kwargs)->plt.Axes:
     "Display `Image` in notebook."
     if ax is None: fig,ax = plt.subplots(figsize=figsize)
-    ax.imshow(image2np(img.data), cmap=cmap, alpha=alpha, **kwargs)
+    if ('cpu' not in dir(img)): ax.imshow(img)
```

Comment:
We should also pass `cmap`, `alpha`, and `kwargs` there I think. Also, `hasattr(imh, 'cpu')` is a little bit cleaner than checking `dir`.